vars/artifactSetVersion.groovy

```diff
@@ -126,10 +124,5 @@ def call(Map parameters = [:]) {
 }
 
 def getTimestamp(pattern){
-    return sh(returnStdout: true, script: "date +'${pattern}'").trim()
+    return sh(returnStdout: true, script: "date -u +'${pattern}'").trim()
```

Comment:
use  long text `--universal`

Reply:
Thanks for the suggestion, I also prefer long options for scripting purposes, not sure why I did not do this to begin with. Updated.